{"pagelen": 100, "values": [{"links": {"self": {"href": "data/repositories/osrf/gazebo/issues/2699/comments/56443488.json"}, "html": {"href": "#!/osrf/gazebo/issues/2699#comment-56443488"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/gazebo/issues/2699.json"}}, "type": "issue", "id": 2699, "repository": {"links": {"self": {"href": "data/repositories/osrf/gazebo.json"}, "html": {"href": "#!/osrf/gazebo"}, "avatar": {"href": "data/bytebucket.org/ravatar/{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}ts=1694483"}}, "type": "repository", "name": "gazebo", "full_name": "osrf/gazebo", "uuid": "{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}"}, "title": "Segfault on conda-build with JPEG parameter struct mismatch: library thinks size is 632, caller expects 664"}, "content": {"raw": "I think that based  on the discussion on [https://github.com/conda-forge/gazebo-feedstock/pull/11](https://github.com/conda-forge/gazebo-feedstock/pull/11), we can close this issue? \n\nTL;DR: It turns out that the problem was that Qt and Freeimage  were linking two different \\(ABI-incompatible\\) versions of libjpeg.", "markup": "markdown", "html": "<p>I think that based  on the discussion on <a data-is-external-link=\"true\" href=\"https://github.com/conda-forge/gazebo-feedstock/pull/11\" rel=\"nofollow\">https://github.com/conda-forge/gazebo-feedstock/pull/11</a>, we can close this issue? </p>\n<p>TL;DR: It turns out that the problem was that Qt and Freeimage  were linking two different (ABI-incompatible) versions of libjpeg.</p>", "type": "rendered"}, "created_on": "2020-03-07T15:05:47.451938+00:00", "user": {"display_name": "Silvio Traversaro", "uuid": "{34f404cb-5642-4f27-a032-e04c7143d776}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B34f404cb-5642-4f27-a032-e04c7143d776%7D"}, "html": {"href": "https://bitbucket.org/%7B34f404cb-5642-4f27-a032-e04c7143d776%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:e1c3e6a0-9a64-4e5f-a668-feebc191c5b6/578caaa1-056c-45a2-8bf9-bee94c3bebef/128"}}, "nickname": "traversaro", "type": "user", "account_id": "557058:e1c3e6a0-9a64-4e5f-a668-feebc191c5b6"}, "updated_on": null, "type": "issue_comment", "id": 56443488}, {"links": {"self": {"href": "data/repositories/osrf/gazebo/issues/2699/comments/56444529.json"}, "html": {"href": "#!/osrf/gazebo/issues/2699#comment-56444529"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/gazebo/issues/2699.json"}}, "type": "issue", "id": 2699, "repository": {"links": {"self": {"href": "data/repositories/osrf/gazebo.json"}, "html": {"href": "#!/osrf/gazebo"}, "avatar": {"href": "data/bytebucket.org/ravatar/{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}ts=1694483"}}, "type": "repository", "name": "gazebo", "full_name": "osrf/gazebo", "uuid": "{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}"}, "title": "Segfault on conda-build with JPEG parameter struct mismatch: library thinks size is 632, caller expects 664"}, "content": {"raw": "yes, thanks again silvio for helping with this! \n\n\u200c\n\nworks fine now locally, and I will take care of rebuilding freeimage on conda-forge!", "markup": "markdown", "html": "<p>yes, thanks again silvio for helping with this! </p>\n<p>\u200c</p>\n<p>works fine now locally, and I will take care of rebuilding freeimage on conda-forge!</p>", "type": "rendered"}, "created_on": "2020-03-07T18:30:42.355621+00:00", "user": {"display_name": "Wolf Vollprecht", "uuid": "{103a3d39-4291-483b-b80a-0fa36b48de0f}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B103a3d39-4291-483b-b80a-0fa36b48de0f%7D"}, "html": {"href": "https://bitbucket.org/%7B103a3d39-4291-483b-b80a-0fa36b48de0f%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/4383545af2875d31a664d5dafa75d537d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsWV-1.png"}}, "nickname": "Wolf", "type": "user", "account_id": "557058:d49ea8c4-184b-403a-b664-8b3fa004cf9c"}, "updated_on": null, "type": "issue_comment", "id": 56444529}, {"links": {"self": {"href": "data/repositories/osrf/gazebo/issues/2699/comments/56444530.json"}, "html": {"href": "#!/osrf/gazebo/issues/2699#comment-56444530"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/gazebo/issues/2699.json"}}, "type": "issue", "id": 2699, "repository": {"links": {"self": {"href": "data/repositories/osrf/gazebo.json"}, "html": {"href": "#!/osrf/gazebo"}, "avatar": {"href": "data/bytebucket.org/ravatar/{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}ts=1694483"}}, "type": "repository", "name": "gazebo", "full_name": "osrf/gazebo", "uuid": "{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}"}, "title": "Segfault on conda-build with JPEG parameter struct mismatch: library thinks size is 632, caller expects 664"}, "content": {"raw": null, "markup": "markdown", "html": "", "type": "rendered"}, "created_on": "2020-03-07T18:30:48.877322+00:00", "user": {"display_name": "Wolf Vollprecht", "uuid": "{103a3d39-4291-483b-b80a-0fa36b48de0f}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B103a3d39-4291-483b-b80a-0fa36b48de0f%7D"}, "html": {"href": "https://bitbucket.org/%7B103a3d39-4291-483b-b80a-0fa36b48de0f%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/4383545af2875d31a664d5dafa75d537d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsWV-1.png"}}, "nickname": "Wolf", "type": "user", "account_id": "557058:d49ea8c4-184b-403a-b664-8b3fa004cf9c"}, "updated_on": null, "type": "issue_comment", "id": 56444530}], "page": 1, "size": 3}